scripts/e2e/ios-devices.js

```diff
@@ -0,0 +1,52 @@
+const DEFAULT_APPLE_DEVICE = 'iPhone 17';
+const DEFEAULT_IOS_VERSION = 'iOS 26.2';
+
+/**
+ * @return {string}
+ */
+function getAppleDevice() {
+  const envVariableKey = 'RNS_E2E_APPLE_SIM_NAME';
+  const passedDevice = process.env[envVariableKey];
+  if (passedDevice) {
+    if (/^(iPhone|iPad)\s.+/.test(passedDevice)) {
+      return passedDevice;
+    } else {
+      throw new Error(`Environment variable ${envVariableKey} should be "iPhone xyz" or "iPad xyz".`);
+    }
+  }
+  return process.env.RNS_E2E_APPLE_SIM_NAME || DEFAULT_APPLE_DEVICE;
+}
+/**
+ * @return {`iOS ${string}`} requested version of ios, or default if not specified
+ */
+function getIOSVersion() {
+  const envVariableKey = 'RNS_E2E_IOS_VERSION';
+  const passedVersion = process.env[envVariableKey];
+  if (passedVersion) {
+    if (/^(iOS)\s.+/.test(passedVersion)) {
```

Comment:
I haven't made up my mind, but we'll have a rather fixed number of checks; therefore, I'm wondering if `startsWith` won't improve readability compared to a regex. Treat it as a nitpick; for me, it's okay to leave it in its current form.